packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/handlers/DeleteFromDiagramEventHandler.java

```diff
@@ -191,7 +193,7 @@ private IStatus invokeDeleteNodeTool(Node node, IEditingContext editingContext,
                 variableManager.put(Node.SELECTED_NODE, node);
                 variableManager.put(DELETION_POLICY, deletionPolicy);
                 variableManager.put(Environment.ENVIRONMENT, new Environment(Environment.SIRIUS_COMPONENTS));
-
+                variableManager.put(IDiagramService.DIAGRAM_SERVICE, new DiagramService(diagramContext));
```

Comment:
Are you sure that it should be `new  DiagramService` here? Why not `new  DiagramServices`?

Reply:
Yes, we want to register an `IDiagramService` variable, on which services from the `IDiagramServices` class can be called (the first parameter of `IDiagramServices` methods is `IDiagramService`).
The names could be reworked though to avoid confusion.